### DISCUSSION TITLE
Fin de combat : luck calculator

### DISCUSSION THREAD

**Discussion by romain22222** [Crownicles/Crownicles#2042] [Suggestions fermées]
Pour faire simple, rajouter un emote à l'embed de résumé de fin de combat, qui quand cliqué dessus te fait apparaître des stats sympas sur le combat, comme en autre : 

- quelle était la proba de faire au moins autant de crits vis à vis des chances respectives de chaque attaque (en gros, est ce que t'es culotté ou pas)
- quelle était la proba que tu rates autant d'attaques ou plus (en gros à quel point tu as été poisseux)
- quelle était la proba classique que tu gagnes (à implémenter bien plus tard si suggestion acceptée, l'élo n'est pas du tout stable pour le moment, les combats encore moins)
- etc.

**Comment by Easygamer01s** [comment]
Hello ;)

Je trouve l'idée vraiment sympa ! Certaines stats vont être très dures à mettre en place, la proba de win devrait être simulée pour être obtenue..

On peut aussi imaginer un ratio de win, nombre de combats par joueur, et la même chose dans pour les combats fait entre les deux joueurs..

**Comment by BastLast** [comment]
tellement long à coder pour si peu d'avantages perso je pense qu'on doit se concentrer sur d'autre priorités

**Reply by romain22222** [reply]
ce serait pas quelquechose qui se coderait en une maj NI un truc codé par un dev majeur, mais ce serait cool d'en avoir un, c'est quelquechose d'original que peu de jeux ont

**Reply by BastLast** [reply]
Le soucis c'est le coût en opportunité

**Reply by Easygamer01s** [reply]
Peut-être juste un compteur du nombre de rencontres avec le même opposant et le pourcentage de victoires ? 😁

Comme bfbc2 avec le nombre de cut ❤️

**Reply by BastLast** [reply]
Plus envisageable déjà.
